### ISSUE TITLE
Why aspnet? [Question]

### ISSUE THREAD

**Issue by AndyPook** [Kymeric/DockerProxy#1]
Just bumped into this. Nice!
I pretty sure that the extra 172. container network is seen as a feature my MS. It does keep some concerns isolated (ie port collisions). And this is really just a dev issue. Any "proper" orchestrator deals with this.

But I'm a dev and this is a pita. So thank you :) But...

Q: Why use aspnetcore at all? Why not a more simple console app that just hosts the ProxyManager?

**Comment by jonstelly** [comment]
Good point... Initially I had thought it might be nice to provide some really simple WebAPI endpoint to get a consumable JSON describing the docker environment, but I don't think that's really needed with the simplicity of the docker CLI.  I'll trim out some fat next time I  get a chance.
